shared/common-adapters/floating-menu.js

```diff
@@ -1,4 +1,11 @@
 // @flow
+
+// For stories, all popups using FloatingMenu will need to have a PropProvider
```

Comment:
This should go in `floating-box` (sry if I told you here heh)

Reply:
Updated to both floating-box desktop and native